build-release-tools/application/jump_version.py

```diff
@@ -146,11 +170,17 @@ def update_package_json(self):
         package_exist = self.package_exist()
         if not package_exist:
             print "No package.json under {0}".format(self._repo_dir)
-            return False
+            return
         try:
+            current_version = NPM.get_current_version(self._repo_dir)
+            # TBD: check whether the commit in manifest is the commit to jump version
+            if current_version == self._version:
+                print "The version of {0} is already {1}".format(self._repo_dir, self._version)
```

Comment:
can you add a "[Warning]" header for this message ?
and says , "The version of {0} is already {1}, skip the version bump action in deb/changelog for {0}"